infrastructure/experiment/main.tf

```diff
@@ -0,0 +1,102 @@
+terraform {
+  required_version = ">= 0.12"
+
+  required_providers {
+    ec = {
+      source  = "elastic/ec"
+      version = "0.1.0-beta"
+    }
+  }
+}
+
+data "terraform_remote_state" "accounts_catalogue" {
+  backend = "s3"
+
+  config = {
+    role_arn = "arn:aws:iam::760097843905:role/platform-read_only"
+
+    bucket = "wellcomecollection-platform-infra"
+    key    = "terraform/platform-infrastructure/accounts/catalogue.tfstate"
+    region = "eu-west-1"
+  }
+}
+
+provider "ec" {
+  apikey = "<api_key>"
+}
+
+provider "aws" {
+  region = "eu-west-1"
+
+  assume_role {
+    role_arn = "arn:aws:iam::756629837203:role/catalogue-developer"
+  }
+}
+
+locals {
+  ec_eu_west_1_service_name = "com.amazonaws.vpce.eu-west-1.vpce-svc-01f2afe87944eb12b"
```

Comment:
These are listed here: https://www.elastic.co/guide/en/cloud/current/ec-traffic-filtering-vpc.html
Should be commented in the TF.